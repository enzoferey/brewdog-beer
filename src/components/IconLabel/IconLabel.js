import React, { Fragment } from "react";
import PropTypes from "prop-types";
import style from "./icon-label.scss";

const IconLabel = ({ icon, label }) => (
  <Fragment>
    <img className={style.icon} src={icon} alt="" />
    <span className={style.label}>{label}</span>
  </Fragment>
);

IconLabel.propTypes = {
  icon: PropTypes.string,
  label: PropTypes.string,
};

export default IconLabel;
