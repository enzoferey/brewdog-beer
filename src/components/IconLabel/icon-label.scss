.icon {
  margin-right: 10px;
}

.label {
  font-size: 18px;
}
