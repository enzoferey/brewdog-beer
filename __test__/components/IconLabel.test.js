import React from "react";
import IconLabel from "components/IconLabel";

const props = {
  icon: "src-path",
  label: "My label",
};

test("renders correctly", () => {
  testSnapshot(<IconLabel {...props} />);
});
