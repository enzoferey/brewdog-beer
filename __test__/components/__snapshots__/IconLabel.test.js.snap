// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <img
    alt=""
    className={undefined}
    src="src-path"
/>,
  <span
    className={undefined}
>
    My label
</span>,
]
`;
